src/Components/Wishlist/Wishlist.css

```diff
@@ -0,0 +1,93 @@
+@import url("../../style.css");
+
+.product-container{
+    height: 34rem;
+    width: 79.8rem;
+    display: flex;
+    justify-content: center;
+    margin-top: 1rem;
+}
+.section-first{
+    height: 25rem;
+    width: 60rem;
+    display: flex;
+    margin-top: 5rem;
+    margin-left: -5rem;
+}
+.heading-four{
+    position: relative;
+    top: -1rem;
+}
+.btn-div{
+    display: flex;
+    column-gap: 1rem;
+    margin-left: 1.5rem;
+}
+.card-content{
+    height: 6.5rem;
+    margin-top: -1.3rem;
+    background-color: var(--sky-blue-color);
+}
+.card-btn{
+    padding: 0.8rem 0.8rem;
```

Comment:
if the padding is the same for all then you can replace it with padding: 0.8rem;